### ISSUE TITLE
Module import line is not beautified

### ISSUE THREAD

**Issue by tell-y** [dai-shi/es-beautifier#1]
It seems that it is not working against `import` line with `max-len` rule.
I'm using at Atom editor.

**Comment by dai-shi** [comment]
can you give me an example, please? input and the expected output would be good. thanks.

**Comment by TeYmmt** [comment]
I expected
```
import { exampleA, exampleB, exampleC, exampleD, exampleE, exampleF, exampleG, exampleH } from 'example';
```
could be 
```
import {
  exampleA,
  exampleB,
  exampleC,
  exampleD,
  exampleE,
  exampleF,
  exampleG,
  exampleH,
} from 'example';
```

However  it works good like this :
```
const example = { exampleA, exampleB, exampleC, exampleD, exampleE, exampleF, exampleG, exampleH };
```
to be
```
const example = {
  exampleA,
  exampleB,
  exampleC,
  exampleD,
  exampleE,
  exampleF,
  exampleG,
  exampleH,
};
```

**Comment by dai-shi** [comment]
Thanks for reporting this! It's not yet supported. Will try.

**Comment by dai-shi** [comment]
Let me change the title.

**Comment by dai-shi** [comment]
BTW, this is not directly related to the `max-len` rule. eslint rule can't read other eslint rules' configs.

**Comment by dai-shi** [comment]
The commit above adds a new rule to make import declaration multiline. However, it doesn't indent lines well. This is due to the indent rule of original eslint.

The relevant issue is https://github.com/eslint/eslint/issues/6813.

**Comment by dai-shi** [comment]
https://github.com/eslint/eslint/pull/7618

**Comment by dai-shi** [comment]
https://github.com/eslint/eslint/pull/7618 is merged now!

**Comment by dai-shi** [comment]
fixed in v1.0.0.
